server/gitlab/api.go

```diff
@@ -313,12 +405,25 @@ func (g *gitlab) GetUnreads(user *UserInfo) ([]*internGitlab.Todo, error) {
 		return nil, err
 	}
 
-	result, _, err := client.Todos.ListTodos(&internGitlab.ListTodosOptions{})
-	if err != nil {
-		return nil, errors.Wrap(err, "can't list todo in GitLab api")
+	var results []*internGitlab.Todo
+	var result []*internGitlab.Todo
+	var resp *internGitlab.Response
+	opt := &internGitlab.ListTodosOptions{
+		ListOptions: internGitlab.ListOptions{Page: 1, PerPage: 1},
+	}
+	for {
+		result, resp, err = client.Todos.ListTodos(opt)
```

Comment:
I think we only need to declare `result` in the upper scope if we write it like this:

```suggestion
		current, resp, err := client.Todos.ListTodos(opt)
```

Notice the colon by the equals sign